src/content/tooling/wrangler/webpack.md

```diff
@@ -89,9 +89,40 @@ module.exports = {
 
 ```js
 module.exports = {
-  "target": "webworker",
-  "entry": "./index.js",
-  "mode": "production"
+  target: "webworker",
+  entry: "./index.js",
+  mode: "production"
+}
+```
+
+### Using with Workers Sites
+
+Wrangler commands are run from the project root, so ensure your `entry` and `context` are set appropriately. For a project with structure:
+
+```console
+.
+├── public
+│   ├── 404.html
```

Comment:
Where's webpack.config.js? I'd remove all the images and just keep the two html files for brevity